mne/viz/evoked.py

```diff
@@ -2046,7 +2047,12 @@ def plot_compare_evokeds(evokeds, picks=None, gfp=False, colors=None,
         callable, it must take as its single argument an array
         (observations x times) and return the upper and lower confidence bands.
         If None or False, no confidence band is plotted.
-        If True, a 95%% bootstrapped confidence interval is drawn.
+        If True, a 95%% bootstrapped confidence interval is drawn, unless
+        ``evokeds`` does not have > 2 Evoked objects for each condition,
+        in which case no CI is drawn.
+        To plot a CI and a single condition, supply a list containing a list
+        containing multiple Evoked objects (like ``[[Evoked1, Evoked2,
+        Evoked3]]``).
```

Comment:
The complaint is about this docstring getting to be too long. Can some of this be moved to a `Notes` section?